config/yaml_config.go

```diff
@@ -75,19 +77,33 @@ func (a *AgentConfig) loadNetworkYamlConfig(path string) error {
 	// Pull additional parameters from the global config file.
 	a.LogLevel = config.Datadog.GetString("log_level")
 	a.StatsdPort = config.Datadog.GetInt("dogstatsd_port")
+	if config.Datadog.IsSet("bind_host") {
```

Comment:
Did this move from somewhere, or was it left out before?

Reply:
It was left out before. Noticed it when moving things over